### DISCUSSION TITLE
Astroid V3.0.2 issues / CSS + Saving

### DISCUSSION THREAD

**Discussion by southpaw3229** [templaza/astroid-framework#433] [General]
First, I really live the new astroid backend as it loads a lot faster.  That being said, i've noticed some weird things.  
1.  Sometimes it doesn't save and just spins
2. I've noticed when I can save, it throws off my CSS for a while and have to keep clearing the cache.  I do have gzip enabled which might be affecting it.

**Comment by sonvnn** [comment]
May be your login session is expired. You need to relogin and reload page. I fixed this issue with Keepalive js. You can download v3 and test it https://github.com/templaza/astroid-framework/archive/refs/heads/v3.zip

**Comment by southpaw3229** [comment]
Thanks and will try that
